### PR TITLE
A CI workflow to test our python packages

### PR DESCRIPTION
**Pull request by ct2034**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            # Set up ROS
+            - name: Set up ROS
+              uses: ros-tooling/setup-ros@v0.7
+              with:
+                required-ros-distributions: ${{ matrix.ros-distro }}
+            # Get smc_storm for testing
+            - name: Get smc_storm
+              id: get_smc_storm
+              run: |
+                wget https://github.com/convince-project/smc_storm/releases/download/0.0.1/smc_storm_executable.tar.gz
```
Comment (MarcoLm993) [inline .github/workflows/test.yml:29]: Why not using latest here?

**Comment by MarcoLm993** [comment]
Looks great! As soon as we have a first BT example running, we could even check if we get the expected result from storm by reading from std::out?

**Review by MarcoLm993 — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (ct2034) [reply .github/workflows/test.yml:29]: I thought it gets more predictable results when fixing the version.

Comment (MarcoLm993) [reply .github/workflows/test.yml:29]: Hmm, I think you have a point here. What about making this more explicit then, adding an entry for the smc_storm version and using it in the wget command afterwards?
The easiest way would be doing everything in bash:
```suggestion
                SMC_STORM_VERSION=0.0.1
                wget https://github.com/convince-project/smc_storm/releases/download/${SMC_STORM_VERSION}/smc_storm_executable.tar.gz
```

**Pull request merged**